File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/YaContestHelperApplication.java
package org.ilyatyamin.yacontesthelper;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.scheduling.annotation.EnableScheduling;
//import springfox.documentation.swagger2.annotations.EnableSwagger2;

@SpringBootApplication()
@EnableFeignClients
@EnableScheduling
//@EnableSwagger2
public class YaContestHelperApplication {
    public static void main(String[] args) {
        SpringApplication.run(YaContestHelperApplication.class, args);
    }
}

File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/YaContestHelperApplication.kt
package org.ilyatyamin.yacontesthelper

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.openfeign.EnableFeignClients
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableFeignClients
@EnableScheduling
open class YaContestHelperApplication

fun main(args: Array<String>) {
    SpringApplication.run(YaContestHelperApplication::class.java, *args)
}
